### PR TITLE
chore(zh): content clean up for markdown-lint-cli bump

### PR DESCRIPTION
**Pull request by yin1999**
### Description

content clean up for markdown-lint-cli bump

### Related issues and pull requests

prepare for: mdn/translated-content#17042

### CODE REVIEW

File: files/zh-tw/web/javascript/reference/global_objects/array/index.md [hunk 1]
```diff
@@ -7,112 +7,9 @@ slug: Web/JavaScript/Reference/Global_Objects/Array
...
 JavaScript 中的 **`Array`** 全域物件被用於建構陣列；陣列為高階（high-level）、似列表（list-like）的物件。陣列在 Javascript 裡面並沒有固定的長度與型別。由於陣列的長度可以隨時被改變，所以並不能保證陣列的密度。這取決於開發者如何使用陣列。一般來說，這是個非常方便的特性，但如果這並不適用於你的開發工作，你也許會考慮使用型別陣列。
 
-**建立陣列**
-
-```js
-var fruits = ["Apple", "Banana"];
-
-console.log(fruits.length);
-// 2
-```
```
Comment (yin1999) [inline files/zh-tw/web/javascript/reference/global_objects/array/index.md:17]: Move to the examples section

File: files/zh-cn/web/api/messageport/index.md [hunk 2]
```diff
@@ -38,18 +38,18 @@ _继承自父类 {{domxref("EventTarget")}} 的事件回调_
   - : 当 `MessagePort` 对象收到无法被反序列化的消息时触发。
     也可以通过 {{domxref("MessagePort.onmessageerror","onmessageerror")}} 属性使用。
 
-## 例子
+## 示例
 
-在下面的例子中，你可以看到一个使用 {{domxref("MessageChannel.MessageChannel","MessageChannel()")}} 构造函数创建出的新 channel.
+在下面的示例中，你可以看到一个使用 {{domxref("MessageChannel.MessageChannel","MessageChannel()")}} 构造函数创建出的新 channel.
 
 当 IFrame 加载完成后，我们给 {{domxref("MessageChannel.port1")}} 注册了一个 {{domxref("MessagePort.onmessage","onmessage")}} 回调，并且使用 {{domxref("window.postMessage")}} 方法把 {{domxref("MessageChannel.port2")}} 和一条消息一起传给 IFrame.
```
Comment (jasonren0403) [inline files/zh-cn/web/api/messageport/index.md:45]:
```suggestion
当 IFrame 加载完成后，我们给 {{domxref("MessageChannel.port1")}} 注册了一个 {{domxref("MessagePort.onmessage","onmessage")}} 回调，并且使用 {{domxref("window.postMessage")}} 方法把 {{domxref("MessageChannel.port2")}} 和一条消息一起传给 IFrame。
```

File: files/zh-cn/web/api/messageport/index.md [hunk 3]
```diff
@@ -38,18 +38,18 @@ _继承自父类 {{domxref("EventTarget")}} 的事件回调_
   - : 当 `MessagePort` 对象收到无法被反序列化的消息时触发。
     也可以通过 {{domxref("MessagePort.onmessageerror","onmessageerror")}} 属性使用。
 
-## 例子
+## 示例
 
-在下面的例子中，你可以看到一个使用 {{domxref("MessageChannel.MessageChannel","MessageChannel()")}} 构造函数创建出的新 channel.
+在下面的示例中，你可以看到一个使用 {{domxref("MessageChannel.MessageChannel","MessageChannel()")}} 构造函数创建出的新 channel.
```
Comment (jasonren0403) [inline files/zh-cn/web/api/messageport/index.md:43]:
```suggestion
在下面的示例中，你可以看到一个使用 {{domxref("MessageChannel.MessageChannel","MessageChannel()")}} 构造函数创建出的新通道。
```

File: files/zh-cn/web/api/navigator/getusermedia/index.md [hunk 4]
```diff
@@ -96,7 +96,7 @@ if (navigator.getUserMedia) {
 
 > **警告：** 新代码应当使用 {{domxref("Navigator.mediaDevices.getUserMedia()")}} 替代。
 
-## 更多参见
+## 参见
```
Comment (jasonren0403) [inline files/zh-cn/web/api/navigator/getusermedia/index.md:100]: L104 should be fixed(taken care of upstream in https://github.com/mdn/content/pull/30491/)

File: files/zh-cn/web/api/websockets_api/writing_websocket_servers/index.md [hunk 5]
```diff
@@ -172,8 +172,6 @@ Think of an extension as compressing a file before e-mailing it to someone. What
 
 > **备注：** 扩展在规范的 5.8, 9, 11.3.2, and 11.4 进行了解释
```
Comment (jasonren0403) [inline files/zh-cn/web/api/websockets_api/writing_websocket_servers/index.md:173]: fix it along

```suggestion
> **备注：** 扩展在规范的第 5.8、9、11.3.2 和 11.4 条进行了解释。
```

File: files/zh-cn/web/api/window/ondragdrop/index.md [hunk 6]
```diff
@@ -9,45 +9,44 @@ slug: Web/API/Window/ondragdrop
...
+```js-nolint
 window.ondragdrop = funcRef;
 window.addEventListener("dragdrop", funcRef, useCapturing);
 ```
 
-- funcRef
+- `funcRef`
   - : 要注册的事件处理函数。
 
 [Gecko](/zh-CN/docs/Web/API/Window/en/Gecko)([Firefox bug 112288](https://bugzil.la/112288)) 中未实现`window.ondragdrop`属性和`ondragdrop`属性，你必须使用`addEventListener`。有关详细信息，请参见[addEventListener](/zh-CN/docs/Web/API/Window/en/DOM/element.addEventListener)。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/window/ondragdrop/index.md:22]:
```suggestion
Firefox 中未实现 `window.ondragdrop` 属性和 `ondragdrop` 属性（[Firefox bug 112288](https://bugzil.la/112288)），你必须使用 `addEventListener`。有关详细信息，请参见 [addEventListener](/zh-CN/docs/Web/API/EventTarget/addEventListener)。
```

File: files/zh-cn/web/http/headers/permissions-policy/index.md [hunk 7]
```diff
@@ -19,18 +19,14 @@ slug: Web/HTTP/Headers/Permissions-Policy
...
-
-- `*`: 本特性默认在最上层和包含的内容中（iframes）允许。
-- `'self'`: 本特性默认在最上层允许，而包含的内容中（iframes）使用源地址相同设定。也就是说本特性在 iframe 中不允许跨域访问。
-- `'none'`: 本特性默认在最上层和包含的内容中（iframes）都禁止。
+    - `*`: 允许在当前文档和所有包含的内容（比如 iframes）中使用本特性。
+    - `'self'`: 允许在当前文档中使用本特性，但在包含的内容（比如 iframes）仍使用原值。
+    - `'src'`: (只在 iframe 中允许) 只要在{{HTMLElement('iframe','src','#Attributes')}} 中的 URL 和加载 iframe 用的 URL 相同，则本特性在 iframe 中允许，
+    - `'none'`: 从最上层到包含的内容都禁止本特性。 \<origin(s)>: 在特定的源中允许，源 URL 以空格分割。
 
 `*`(在所有源地址启用)`或'none'`(在所有源地址禁用) 只允许单独使用，而`'self'`和`'src'`可以与多个源地址一起使用。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/headers/permissions-policy/index.md:31]:
```suggestion
`*` 或 `none` 值只允许单独使用，而 `self` 和 `src` 值可以与多个源一起使用。
```

File: files/zh-cn/web/javascript/reference/global_objects/string/replace/index.md [hunk 8]
```diff
@@ -154,7 +154,7 @@ console.log(newstr); // Cruz, Maria
 
 这将打印 `'Cruz, Maria'`。
 
-### 使用內联函数来修改匹配到的字符。
+### 使用內联函数来修改匹配到的字符
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/string/replace/index.md:157]:
```suggestion
### 使用内联函数来修改匹配到的字符
```

File: files/zh-tw/learn/css/howto/css_faq/index.md [hunk 9]
```diff
@@ -89,24 +92,23 @@ CSS 並不支援「以另一個樣式規則為基準」的樣式設定法。 (
 
 你可以使用 [DOM 觀察器](zh_tw/DOM_%e8%a7%80%e5%af%9f%e5%99%a8) 的 _CSS Style Rules_ 來檢查上述問題。
```
Comment (jasonren0403) [inline files/zh-tw/learn/css/howto/css_faq/index.md:93]: possible link to this

```suggestion
你可以使用 [DOM 觀察器](https://firefox-source-docs.mozilla.org/devtools-user/page_inspector/how_to/examine_and_edit_css/index.html) 的 _CSS Style Rules_ 來檢查上述問題。
```

File: files/zh-cn/web/api/navigator/getusermedia/index.md (hunk 4 above)
Comment (yin1999) [reply files/zh-cn/web/api/navigator/getusermedia/index.md:100]: fixed

**Review by jasonren0403 — APPROVED** [review]

**Pull request merged**